vignettes/ChromBand.Rmd

```diff
@@ -0,0 +1,437 @@
+---
+title: "Using Chromosome Bands as Categories"
+author: 
+   - name: D. Sarkar
+   - name: S. Falcon
+   - name: R. Gentleman
+   - name: Kemmy MO Jones
+     affiliation: [ Chromosome Bands Vignette translation from Sweave to R Markdown / HTML ]
```

Comment:
Remove the `[` and `]`.

Reply:
ok

Reply:
To clarify, I am asking you to remove the brackets in `[ Chromosome Bands Vignette translation from Sweave to R Markdown / HTML ]` so that it is `Chromosome Bands Vignette translation from Sweave to R Markdown / HTML`.